hoomd/ParticleGroup.cc

```diff
@@ -789,6 +792,100 @@ unsigned int ParticleGroup::intersectionSize(std::shared_ptr<ParticleGroup> othe
     return n;
     }
 
+void ParticleGroup::thermalizeParticleMomenta(Scalar kT, unsigned int seed, unsigned int timestep)
+    {
+    unsigned int group_size = this->getNumMembers();
+
+    const unsigned int D = Scalar(m_sysdef->getNDimensions());
```

Comment:
I would prefer the use of `dimensions` instead of `D`, easy to forget the meaning of single letter variables later in the code.

Reply:
Done.